SwordImportExportPlugin.inc.php

```diff
@@ -0,0 +1,230 @@
+				$depositEditorial = $request->getUserVar('depositEditorial');
+				$depositGalleys = $request->getUserVar('depositGalleys');
+				$username = $request->getUserVar('swordUsername');
+				$depositIds = array();
+
+				$backLink = $request->url(
+					null, null, null,
+					array('plugin', $this->getName()),
+					array(
+						'selectedDepositPoint' => $depositPointId,
+						'depositEditorial' => $depositEditorial,
+						'depositGalleys' => $depositGalleys,
+					)
+				);
+
+				$errors = array();
+				// select at least one article
+				$articleIds = $request->getUserVar('articleId');
+				if (empty($articleIds)) {
+					$errors[] = array(
+						'title' => __('plugins.importexport.sword.requiredFieldErrorTitle'),
+						'message' => __('plugins.importexport.sword.requiredFieldErrorMessage'),
+					);
+				}
+				else {
+					foreach ($articleIds as $articleId) {
+						$publishedArticle = $publishedArticleDao->getByArticleId($articleId);
+						try {
+							$deposit = new OJSSwordDeposit($publishedArticle);
+							$deposit->setMetadata($request);
+							if ($depositGalleys) $deposit->addGalleys();
+							if ($depositEditorial) $deposit->addEditorial();
+							$deposit->createPackage();
+							$response = $deposit->deposit($swordDepositPoint, $username, $password);
+							$deposit->cleanup();
+							$depositIds[] = $response->sac_id;
+						}
+						catch (Exception $e) {
+							$errors[] = array(
+								'title' => $publishedArticle->getLocalizedTitle(),
+								'message' => $e->getMessage(),
+							);
+						}
+					}
+				}
+
+				if (!empty($errors)) {
+					$errorMessage = '<dl>';
+					foreach ($errors as $error) {
+						$errorMessage .= "<dt>{$error['title']}</dt>";
+						$errorMessage .= "<dd>{$error['message']}</dd>";
+					}
+					$errorMessage .= '</dl>';
+					$templateMgr->assign(array(
+						'title' => __('plugins.importexport.sword.depositFailed'),
+						'messageTranslated' => $errorMessage,
+						'backLink' => $backLink,
+						'backLinkLabel' => 'common.back'
+					));
+				}
+				else {
+					$templateMgr->assign(array(
+						'title' => __('plugins.importexport.sword.depositSuccessful'),
+						'message' => 'plugins.importexport.sword.depositSuccessfulDescription',
+						'backLink' => $backLink,
+						'backLinkLabel' => 'common.continue'
+					));
+				}
+				$messageTemplateFile = $this->getSwordPlugin()->getTemplatePath() . 'message.tpl';
+				$output = $templateMgr->fetch($messageTemplateFile);
+				return new JSONMessage(true, $output);
+				break;
+			default:
+				$dispatcher = $request->getDispatcher();
+				$dispatcher->handle404();
+		}
+	}
+
+	/**
+	 * @copydoc PKPImportExportPlugin::executeCLI()
+	 */
+	public function executeCLI($scriptName, &$args) {
+		throw new Exception('import/export tasks using the command-line not yet implemented!');
+	}
+
+	/**
+	 * @copydoc PKPImportExportPlugin::usage
+	 */
+	public function usage($scriptName) {
```

Comment:
If CLI import/export isn't supported, there's no point in giving usage directions.